sources/packages/backend/apps/db-migrations/src/sql/Queue/Update-disbursement-receipts-file-cron.sql

```diff
@@ -0,0 +1,11 @@
+update
+    sims.queue_configurations
+set
+    queue_configuration = jsonb_set(
+        queue_configuration,
+        '{cron}',
+        '"30 18 * * 1 -5"',
```

Comment:
Please remove the space in between `1 -5`. When I evaluate the expression online, the space breaks the cron. 
![image](https://github.com/user-attachments/assets/613dd9ff-206b-4241-ad58-786526da82fc)